Modules/mmapmodule.c

```diff
@@ -1244,8 +1253,8 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
                                 "tagname",
                                 "access", "offset", NULL };
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|ziL", keywords,
-                                     &fileno, &map_size,
+    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|ziL:mmap.__new__",
```

Comment:
I can't think of a simple way to achieve that, as the overflow happens inside PyArg_ParseTupleAndKeywords, but it could happen in 4 different arguments, so new_mmap_object can't know in which of the arguments the overflow happened.

that's why I made the patch in seterror(). that was the best simple solution I could come up with.